src/main/java/org/opensrp/web/GlobalExceptionHandler.java

```diff
@@ -42,6 +45,22 @@
         return buildErrorResponse(HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
+    @ResponseBody
+    @ExceptionHandler(UploadValidationException.class)
```

Comment:
This status code does not seem correct https://httpstatuses.com/417 is for use when the server does understand  accept headers sent from client, this should be a bad request